RecoLocalTracker/SiPhase2Clusterizer/test/ClustersValidation.cc

```diff
@@ -33,6 +33,8 @@
 #include "SimDataFormats/Vertex/interface/SimVertexContainer.h"
 #include "SimDataFormats/TrackingHit/interface/PSimHitContainer.h"
 
+#include "SimTracker/SiPhase2Digitizer/interface/Phase2TrackerDigiCommon.h"
```

Comment:
Do I understand correctly that your comment is related to the file in SimTracker and not the ClusterValidation file ? If so, I cannot move this file but can copy/paste the code locally (which might not be the best solution).